docs/getting-started/README.md

```diff
@@ -1,7 +1,13 @@
 # Getting started with the Forms System Library
 
-### Creating a new application with the US Forms System Starter App
+Depending on your needs, you can either create a brand new application from scratch or install the US Forms Library in an existing application.
```

Comment:
This is nitpicky, but I notice sometimes we refer to this library as US Forms System, or US Forms Library, or Forms System Library. We might just want to pick 1 name and be consistent.

Reply:
Yes, we definitely need to standardize. Will poll on which we like. 👍

Reply:
Per conversation in HipChat, we're going to go with US Forms System.